buildpipeline/Core-Setup-Signing-Windows-x64.json

```diff
@@ -477,7 +477,10 @@
     "GITHUB_PASSWORD": {
       "value": "PassedViaPipeBuild"
     },
-    "SIGNED_PACKAGES": {
+    "BUILD_FULL_PLATFORM_MANIFEST": {
+      "value": "true"
+    },
+    "PUBLISH_RID_AGNOSTIC_PACKAGES": {
       "value": "true"
```

Comment:
I think I see the scenario this prepares us for now: if we need another pipebuild leg that uses the Core-Setup-Signing-Windows-x64 definition, then we can enable `BUILD_FULL_PLATFORM_MANIFEST` for both, but only enable `PUBLISH_RID_AGNOSTIC_PACKAGES` for one. Thanks.